tests/EventTest.php

```diff
@@ -91,6 +92,7 @@ public function testEventIsGeneratedWithUniqueIdentifier(): void
     public function testToArray(): void
     {
         $this->options->setRelease('1.2.3-dev');
+        $sentryPrettyVersion = PrettyVersions::getVersion('sentry/sentry')->getPrettyVersion();
```

Comment:
This can be inlined directly in the array, there is no reason to declare a temporary variable to hold a value used only in one place